autogluon/utils/try_import.py

```diff
@@ -48,3 +48,20 @@ def try_import_cv2():
         raise ImportError(
             "Unable to import dependency cv2. "
             "A quick tip is to install via `pip install opencv-python`. ")
+
+def try_import_gluonnlp():
+    try:
+        import gluonnlp
+        # TODO After 1.0 is supported,
+        #  we will remove the checking here and use gluonnlp.utils.check_version instead.
+        from pkg_resources import parse_version  # pylint: disable=import-outside-toplevel
+        gluonnlp_version = parse_version(gluonnlp.__version__)
+        assert gluonnlp_version >= parse_version('0.8.1') and\
+               gluonnlp_version <= parse_version('0.8.3'), \
+            'Currently, we only support 0.8.1<=gluonnlp<=0.8.3'
+    except ImportError:
+        raise ImportError(
+            "Unable to import dependency gluonnlp. The NLP model won't be available "
+            "without installing gluonnlp. "
+            "A quick tip is to install via `pip install gluonnlp`. ")
```

Comment:
change to `pip install gluonnlp==0.8.1`

Reply:
Also add a comment to the text tutorial telling the user to `pip install gluonnlp==0.8.1` prior to running.